src/repository/resources/ipres.rs

```diff
@@ -1709,12 +1709,128 @@ impl AddressFamily {
 }
 
 
+//------------ Ipv4Block -----------------------------------------------------
+
+/// Contains an IPv4 address block.
```

Comment:
This self-referential description doesn't really tell you anything. Would it perhaps be an improvement to base the description on that of "IpBlock", e.g. to say:

```suggestion
/// A consecutive sequence of IPv4 addresses.
```

?